### PR TITLE
Fixed NullPointer Exception in Android 5 due to invalid column name on cursor

### PR DESCRIPTION
**Pull request by aroberson**
http://stackoverflow.com/questions/30640251/cordova-resolvelocalfilesystemurl-error-code-1000-in-android/30923570#30923570

Was getting this when trying to retrieve the last modified date.

### CODE REVIEW

**Comment by cjpearson** [comment]
@infil00p, could you take a look at this PR? I'm running into this issue when I try to open a file that was taken from the camera plugin on Android Lollipop.

**Comment by didlich** [comment]
+1

**Comment by infil00p** [comment]
Can you create a JIRA issue for this, so I know what I'm testing?

**Comment by cjpearson** [comment]
https://issues.apache.org/jira/browse/CB-9965

File: src/android/ContentFilesystem.java [hunk 1]
```diff
@@ -18,22 +18,19 @@ Licensed to the Apache Software Foundation (ASF) under one
...
 import android.net.Uri;
+import android.provider.DocumentsContract;
 import android.provider.MediaStore;
 import android.provider.OpenableColumns;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import org.apache.cordova.CordovaResourceApi;
+import org.json.JSONException;
+import org.json.JSONObject;
```
Comment (jasongin) [inline src/android/ContentFilesystem.java]: Please keep the import statements separated into groups. Ideally I think they should follow the [Android style](http://source.android.com/source/code-style.html#order-import-statements), but not all of Cordova's Android code follows that now.

**Comment by jasongin** [comment]
Looks good to me other than the minor style issue.

File: src/android/ContentFilesystem.java (hunk 1 above)
Comment (infil00p) [inline src/android/ContentFilesystem.java]: LULZ! They follow the "Android Studio/Eclipse put it there for me style".

Seriously, Cordova, Android and Java imports should be grouped separately, but I'm not completely concerned about the order of the groupings.

**Comment by stevengill** [comment]
I think we need @aroberson to sign and submit an ICLA https://www.apache.org/licenses/icla.txt

**Comment by aroberson** [comment]
I will get to this tonight.

**Comment by aroberson** [comment]
I filled it out and sent it to the secretary.

Thanks,

Adam Roberson

**Comment by infil00p** [comment]
Looks good here.  Had to spend more time fixing camera issues than actually testing this PR. :/

**Pull request closed without merging**